ui: add store disk write bytes to storage dashboard

annrpom opened the pull request:
This patch adds a metric for store disk write bytes in the Storage dashboard.

Epic: none
Informs: https://github.com/cockroachlabs/support/issues/3470

Release note: None

<img width="720" height="308" alt="image" src="https://github.com/user-attachments/assets/94629b53-2dc7-4a42-8b68-85ddbd93d006" />
<img width="968" height="411" alt="image" src="https://github.com/user-attachments/assets/e7136191-8df7-4dcf-8150-815661395af1" />

<img width="968" height="407" alt="image" src="https://github.com/user-attachments/assets/3add06df-0f76-48c5-b467-aca2f9742869" />
<img width="975" height="414" alt="image" src="https://github.com/user-attachments/assets/0b6b5812-be09-4def-83e4-96ce57dd36de" />

annrpom committed:
ui: add store disk write bytes to storage dashboard

This patch adds a metric for store disk write bytes in
the Storage dashboard.

Epic: none

Release note: None

jbowens reviewed (approved):
lgtm!

sumeerbhola reviewed (commented):
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/156970)* status: <span>:shipit:</span> complete! 0 of 0 LGTMs obtained (waiting on @angles-n-daemons)

___
*[`pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/storage.tsx` line 486 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/156970#-OdLuzrgDdGcDYKCgB_Y:-OdLuzrgDdGcDYKCgB_Z:b-zg29bn) ([raw file](https://github.com/cockroachdb/cockroach/blob/187fad1613e1f001eef3339126c6b75fa6bde885/pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/storage.tsx#L486)):*
> ```XML
>         {storeMetrics(
>           {
>             name: "cr.store.storage.disk.write.bytes",
> ```

how about adding disk.read.bytes too.


<!-- Sent from Reviewable.io -->

annrpom committed:
ui: add store disk read bytes to storage dashboard

This patch adds a metric for store disk read bytes in
the Storage dashboard.

Epic: none

Release note: None

annrpom reviewed (commented):
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/156970)* status: <span>:shipit:</span> complete! 0 of 0 LGTMs obtained (waiting on @angles-n-daemons and @sumeerbhola)

___
*[`pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/storage.tsx` line 486 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/156970#-OdLuzrgDdGcDYKCgB_Y:-OdPoshO2x-LSllnyLzb:b4f4vu0) ([raw file](https://github.com/cockroachdb/cockroach/blob/3e8bf52dcb8384c4f68fea1e6cbae221b2df34f6/pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/storage.tsx#L486)):*
<details><summary><i>Previously, sumeerbhola wrote…</i></summary><blockquote>

how about adding disk.read.bytes too.
</blockquote></details>

ok; done


<!-- Sent from Reviewable.io -->

sumeerbhola reviewed (approved):
thanks

annrpom commented:
np; TFTRs! ('-')7

bors r+

annrpom commented:
bors retry
